vuepress/docs/next/docs/curate/ecr-bundle-details.md

```diff
@@ -105,28 +105,55 @@ Here is an example of a plugin descriptor:
 
 **Plugin descriptor.yaml**
 
-    image: "entando/my-image:1.0.0" # The Docker image used to create the plugin
-    deploymentBaseName: "myplugin" # The base name to assign to pods that have to be created in Kubernetes
-    dbms: "postgresql" # The DBMS the plugin will use
-    roles: # The roles the plugin will expose in Keycloak
+    
+    # Descriptor version field added in v3 and later. 
+    # To take advantage of the environment variables below, v4 will need to be specified.
+    descriptorVersion: "v4" 
+
+    # The Docker image used to create the plugin
+    image: "entando/your-image" 
+
+    # The base name to assign to pods that have to be created in Kubernetes
+    deploymentBaseName: "yourplugin" 
+    
+    # The DBMS the plugin will use
+    dbms: "postgresql" 
+
+    # The roles the plugin will expose in Keycloak
+    roles: 
       - "task-list"
       - "task-get"
       - "connection-list"
       - "connection-get"
       - "connection-create"
       - "connection-delete"
       - "connection-edit"
-    healthCheckPath: "/actuator/health" # The health check path that Kubernetes will use to check the status of the plugin deployment
-    ingressPath: "/myhostname.io/entando-plugin" # the ingress path to assign to the plugin deployment
-    permissions: # a list of Keycloak clientIds / roles to bind to one another
+
+    # The health check path that Kubernetes will use to check the status 
+    # of the plugin deployment  
+    healthCheckPath: "/actuator/health" 
+    
+    # The ingress path to assign to the plugin deployment. 
+    ingressPath: "/yourhostname.io/entando-plugin" 
```

Comment:
This was wrong in the original version.  It should not include the hostname.  It should just be 'ingressPath: "/your-plugin-path"